debian/context/etc/sudoers

```diff
@@ -0,0 +1,31 @@
+#
+# This file MUST be edited with the 'visudo' command as root.
+#
+# Please consider adding local content in /etc/sudoers.d/ instead of
+# directly modifying this file.
+#
+# See the man page for details on how to write a sudoers file.
+#
+Defaults	env_reset
+Defaults	mail_badpass
+Defaults	secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+# CIS 1.3.2 - ensure sudo commands use pty
+Defaults    use_pty
+# CIS 1.3.3 - ensure sudo log file exists
+Defaults    logfile='/var/log/sudo.log'
```

Comment:
This will lead to:

sudo: /etc/sudoers:15:21: values for "logfile" must start with a '/'